### DISCUSSION TITLE
How to get a list of all tables as a type?

### DISCUSSION THREAD

**Discussion by sachinraja** [prisma/prisma#8049] [Q&A] [answered]
I'm trying to type a function so it accepts a table from Prisma, but I'm unsure on how I can get a list of the tables as a type. Is there a way to do this without omitting all other properties on `PrismaClient`?

**Comment by ryands17** [comment, marked as answer]
That would be the only way to implement this at the moment. It would be great if you could add a 👍 and go through this request as it is similar to your use case: https://github.com/prisma/prisma/issues/5273

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
